safehttp/plugins/coop/coop.go

```diff
@@ -49,18 +49,26 @@ func (p Policy) String() string {
 	return string(p.Mode) + `; report-to "` + p.ReportingGroup + `"`
 }
 
-// NewInterceptor constructs an interceptor that applies the given policies.
-func NewInterceptor(policies ...Policy) Interceptor {
-	var rep []string
-	var enf []string
+type serializedPolicies struct {
+	rep []string
+	enf []string
+}
+
+func serializePolicies(policies ...Policy) serializedPolicies {
+	var s serializedPolicies
 	for _, p := range policies {
 		if p.ReportOnly {
-			rep = append(rep, p.String())
+			s.rep = append(s.rep, p.String())
 		} else {
-			enf = append(enf, p.String())
+			s.enf = append(s.enf, p.String())
 		}
 	}
-	return Interceptor{rep: rep, enf: enf}
+	return s
+}
+
+// NewInterceptor constructs an interceptor that applies the given policies.
+func NewInterceptor(policies ...Policy) Interceptor {
```

Comment:
Why is `New` as function name not enough?

Reply:
This package exports other types than just Interceptor, so I just added the extra bit to make sure ppl did not confuse this for an overrider or smth else.